symmer/operators/anticommuting_op.py

```diff
@@ -127,12 +127,47 @@ def unitary_partitioning(self, s_index: int=None, up_method: Optional[str]='LCU'
             gamma_l (float): normalization constant of clique (anticommuting operator)
             AC_op (AntiCommutingOp): normalized clique - i.e. self == gamma_l * AC_op
         """
+        assert up_method in ['LCU', 'seq_rot'], f'unknown unitary partitioning method: {up_method}'
         AC_op = self.copy()
+
         if AC_op.n_terms == 1:
-            rotations = None
             gamma_l = np.linalg.norm(AC_op.coeff_vec)
             AC_op.coeff_vec = AC_op.coeff_vec / gamma_l
-            Ps = AC_op
+
+            if AC_op.coeff_vec[0]<0:
```

Comment:
Need to check this sign correction (multiplication fix must commute with global symmerties in CS method!)